chapters/annotations.tex

```diff
@@ -8,6 +8,12 @@ \chapter{Annotations}\label{annotations}
 (For replaceable class declarations with a \lstinline[language=grammar]!constraining-clause! also refer to \cref{constraining-clause-annotations}.)
 The specification in this document defines the semantic meaning if a tool implements any of these annotations.
 
+\section{How to Interpret Annotation Definitions}\label{how-to-interpret-annotation-definitions}
```

Comment:
It can be made clearer that this is a section on notation:
```suggestion
\section{Notation for Annotation Definitions}\label{how-to-interpret-annotation-definitions}

```

Reply:
"Notation for Annotation" It is a bit of a mouthful but I have no better suggestion.